src/openforms/appointments/tests/test_views.py

```diff
@@ -398,8 +399,8 @@ def test_good_token_and_submission_redirect_and_add_submission_to_session(self):
         # after initiating change, we expect the bsn to be stored in plain text (again)
         self.assertEqual(new_submission.auth_info.value, "000000000")
         expected_redirect_url = (
-            f"http://maykinmedia.nl/myform/stap/{form_definition.slug}"
-            f"?submission_uuid={new_submission.uuid}"
+            f"http://maykinmedia.nl/myform/?_of_action=stap&_of_action_args%3D{form_definition.slug}"
+            f"&submission_uuid={new_submission.uuid}"
         )
```

Comment:
perhaps we can clean up this URL building to use `furl` if we're touching the code anyway? That probably makes the "double URLencoding" of the action_args more clear.